src/company_cmd.cpp

```diff
@@ -233,8 +233,8 @@ void InvalidateCompanyWindows()
 		}
 		SetWindowWidgetDirty(WC_COMPANY, cid, WID_C_DESC_COMPANY_VALUE);
 	}
-	_dirty_company_finances = {};
-}
+	_dirty_company_finances.Reset();
```

Comment:
If it's being invalidated every millisecond, does it really need to be manually dirtied here? Or is this just for consistency?

Reply:
This particular line doesn't change any behaviour, I just think `Reset()` is clearer than assigning a new default `CompanyMask`, which also has nothing set.

It's reset so that everything *isn't* invalidated every tick.